borg/archiver.py

```diff
@@ -1399,7 +1440,7 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='ARCHIVE',
                                type=location_validator(archive=True),
                                help='name of archive to create (must be also a valid directory name)')
-        subparser.add_argument('paths', metavar='PATH', nargs='+', type=str,
+        subparser.add_argument('paths', metavar='PATH', nargs='*', type=str,
```

Comment:
Here is a minimal example that illustrates the problem:

```python
    import argparse
    parser = argparse.ArgumentParser()
    parser.add_argument('--foo', action='store_true')
    parser.add_argument('location')

    # works:
    # parser.add_argument('paths', nargs='+')
    # doesn't work:
    parser.add_argument('paths', nargs='*')
    
    args = parser.parse_args('1 --foo 2'.split())
```
With `nargs='*'`, this raises `unrecognized arguments: 2`